docs/input-behaviors.md

```diff
@@ -0,0 +1,24 @@
+# Input behaviors ⌨
+
+The `flame_behaviors` package also provides input behaviors. These behaviors are a layer over the
+existing Flame input mixins for components. These behaviors will trigger when their parent entity
+is being interacted with by the user. So these events are always relative to the parent entity.
+
+## DraggableBehavior
+
+The `DraggableBehavior` brings the [drag events][flame_drag_docs] from Flame into a behavioral
+format.
```

Comment:
```suggestion
The `DraggableBehavior` brings the [drag events][flame_drag_docs] from Flame into a behavioral
format.
```

but what does it mean?

Reply:
That the `DraggableBehavior` brings the drag events from Flame into a behavioral format!

Will try to add a better context/explanation to it!